docs/specs/servers/verify/verify-server-api.md

```diff
@@ -1,30 +1,169 @@
 # Verify Server API
 
+## Generate Challenge (iOS only)
+
+Generates a unique one-time challenge to be later used with other requests as a mean to prevent replay attacks.
+
+### Request
+
+`POST /challenge`
```

Comment:
why not GET?

Reply:
`GET` shouldn't generally be used for something that creates a resource on the backend. The main consideration is caching.